imsim/lsst_image.py

```diff
@@ -145,16 +288,20 @@ def addNoise(self, image, config, base, image_num, obj_num, current_var, logger)
             image_num:      The current image number.
             obj_num:        The first object number in the image.
             current_var:    The current noise variance in each postage stamps.
-            logger:         If given, a logger object to log progress.
+            logger:         A logger object to log progress.
         """
         base['current_noise_image'] = base['current_image']
 
         if logger.isEnabledFor(logging.INFO):
             skyCoord = base['world_center']
+            # Normally skyCoord is a CelestialCoord, but if not make sure this doesn't bork.
```

Comment:
I'm curious:  When would `skyCoord` not be a `CelestialCoord`?

Reply:
It won't be a CelestialCoord if the WCS is not a CelestialWCS.  E.g. in the unit tests for this, I just did a PixelScale wcs, rather than something realistic, which failed the way it had been written.